libs/cart/src/reducers/cart-order/cart-order.reducer.spec.ts

```diff
@@ -39,13 +39,16 @@ describe('Cart | Reducer | CartOrder', () => {
         loading: true
       }
 
-      const cartPlaceOrderSuccess = new DaffCartPlaceOrderSuccess({id: orderId});
+      const cartPlaceOrderSuccess = new DaffCartPlaceOrderSuccess({
+        orderId,
+        cartId: 'cartId',
+      });
 
       result = reducer(state, cartPlaceOrderSuccess);
     });
 
     it('should set the order ID from action.payload', () => {
```

Comment:
This should probably change to "should set the order result from action.payload"

Reply:
changed